src/rules/abapdoc.ts

```diff
@@ -57,6 +54,9 @@ export class Abapdoc extends ABAPRule {
 
     for (const method of methods) {
       const previousRow = method.getStart().getRow() - 2;
+      if (method.isRedefinition()) {
```

Comment:
super pedantic, but you can put this check before the previousRow assignment, looks good otherwise 👍

Reply:
Yes, it should be paid attention to. Now I have changed the statements position.